test/Nancy.Tests/project.json

```diff
@@ -1,6 +1,8 @@
 {
+    "version": "1.4.1.0",

```

Comment:
I'm not entirely sure where to go from this comment. Are you suggesting to leave the code as is for now and then someone (you, me, @jchannon, a roving Samaritan) will "fix" the test separately?